charts/shopware/Chart.lock

```diff
@@ -5,6 +5,15 @@ dependencies:
 - name: operator
   repository: https://shopware.github.io/helm-charts/
   version: 0.0.13
+- name: valkey
+  repository: oci://registry-1.docker.io/bitnamicharts
+  version: 1.0.1
+- name: valkey
+  repository: oci://registry-1.docker.io/bitnamicharts
+  version: 1.0.1
+- name: valkey
+  repository: oci://registry-1.docker.io/bitnamicharts
+  version: 1.0.1
```

Comment:
the three lines are the same looks wrong

Reply:
Probably due to this:
```
  - name: valkey
    alias: valkeyapp
    version: 1.0.1
    repository: oci://registry-1.docker.io/bitnamicharts
    condition: valkeyapp.enabled

  - name: valkey
    alias: valkeysession
    version: 1.0.1
    repository: oci://registry-1.docker.io/bitnamicharts
    condition: valkeysession.enabled

  - name: valkey
    alias: valkeyworker
    version: 1.0.1
    repository: oci://registry-1.docker.io/bitnamicharts
    condition: valkeyworker.enabled
    ```

I guess Helm automatically updates the lock file for each chart imported.

Reply:
Its the lock file. We use aliases for the charts. Check redis.